lib/src/models/comment_impl.dart

```diff
@@ -139,25 +158,172 @@ class MoreComments extends RedditBase {
   }
 }
 
-class Comment extends CommentRef {
+/// A fully initialized class which represents a single Reddit comment.
+class Comment extends CommentRef
```

Comment:
It's probably just me, but I can't seem to find the constructor for this class.

Reply:
It's wayyyyyyy below the properties :)